docs/source/writing_config.rst

```diff
@@ -145,24 +152,26 @@ We must :class:`~fab.steps.analyse.Analyse` the source code to determine the For
     from fab.steps.preprocess import fortran_preprocessor
 
     config = BuildConfig(
-        project_label='my project',
+        project_label='<project label>',
         steps=[
-            GrabFolder(src='~/my_repo'),
+            GrabFolder(src='<program name>'),
             FindSourceFiles(),
             fortran_preprocessor(),
-            Analyse(root_symbol='my_program'),
+            Analyse(root_symbol='<program>'),
```

Comment:
A note about the alternative approach where you want all programs to be discovered and built.